frontend/src/forms/manage-restaurant-form/DetailsSection.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { useFormContext } from 'react-hook-form';
+
+const DetailSection = () => {
+  const { control } = useFormContext();
+  return (
+    <div className="space-y-2">
+      <div>
+        <h2 className="text-2xl font-bold">Details</h2>
+        <FormDescription>Enter details about your restaurant</FormDescription>
+      </div>
+      <FormField
+        control={control}
+        name="restaurantName"
+        render={({ field }) => (
+          <FormItem>
+            <FormLabel>Name</FormLabel>
+            <FormControl>
+              <Input {...field} className="bg-white" />
+            </FormControl>
+            <FormMessage />
+          </FormItem>
+        )}
+      />
+
+      <div className="flex gap-4">
+        <FormField
+          control={control}
+          name="city"
+          render={({ field }) => (
+            <FormItem className="flex-1">
+              <FormLabel>City</FormLabel>
+              <FormControl>
+                <Input {...field} className="bg-white" />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+
+        <FormField
+          control={control}
+          name="country"
+          render={({ field }) => (
+            <FormItem className="flex-1">
+              <FormLabel>Country</FormLabel>
+              <FormControl>
+                <Input {...field} className="bg-white" />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+      </div>
+
+      <FormField
+        control={control}
+        name="deliveryPrice"
+        render={({ field }) => (
+          <FormItem className="max-w-[25%]">
+            <FormLabel>Delivery Price ($)</FormLabel>
+            <FormControl>
+              <Input {...field} className="bg-white" placeholder="1.5" />
+            </FormControl>
+            <FormMessage />
+          </FormItem>
+        )}
+      />
+
+      <FormField
+        control={control}
+        name="estimatedDeliveryTime"
+        render={({ field }) => (
+          <FormItem className="max-w-[25%]">
+            <FormLabel>Estimated Delivery Time (minutes)</FormLabel>
+            <FormControl>
+              <Input {...field} className="bg-white" placeholder="30" />
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/28#issuecomment-2385686684>#fd9344</a><span> - 10/01/2024, 12:51 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The Input component for estimatedDeliveryTime doesn't specify the input type, which may lead to incorrect data entry. A similar issue was also found in frontend/src/forms/manage-restaurant-form/DetailsSection.tsx (line 71).
</div>
<div id="fix">
<b>Fix:</b> Add the type="number" attribute to the Input component for estimatedDeliveryTime to ensure only numeric input is accepted.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -82,7 +82,7 @@
        <FormItem className="max-w-[25%]">
          <FormLabel>Estimated Delivery Time (minutes)</FormLabel>
          <FormControl>
 -          <Input {...field} className="bg-white" placeholder="30" />
 +          <Input {...field} type="number" min="1" step="1" className="bg-white" placeholder="30" />
          </FormControl>
          <FormMessage />
        </FormItem>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged